src/themes/horse-horse-horse.css

```diff
@@ -0,0 +1,211 @@
+:root {
+	--text-black: #474241;
+}
+
+* {
+	box-sizing: border-box;
+}
+
+html,
+body {
+	overflow: hidden;
+	height: 100vh;
+	padding: 0;
+	margin: 0;
+	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica,
+		Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+}
+
+[data-twitch-chat] {
+	--chat-font-size: 16px;
+	--chat-background-color: transparent;
+	--chat-text-color: var(--text-black);
+
+	background-color: var(--chat-background-color);
+	color: var(--chat-text-color);
+	font-size: var(--chat-font-size);
+	width: 99vw;
+	height: 98vh;
+	filter: drop-shadow(0 3px 3px rgb(0 0 0 / 20%));
+	padding: 0.5rem 1rem;
+	border-radius: 20px;
+	list-style: none;
+	margin: 0;
+	overflow: auto;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-end;
+}
+
+[data-twitch-message] {
+	margin: 0.25rem;
+	position: relative;
+	padding-left: 60px;
+}
+
+[data-twitch-message-reply] {
+	display: none;
+}
+
+[data-twitch-emote] {
+	display: inline-block;
+	height: 1.5em;
+	margin-bottom: -0.3em;
+}
+
+[data-twitch-emote] + [data-twitch-emote] {
+	margin-left: -0.22em;
+}
+
+/* Speech Bubble */
+
+.twitch-chat-message,
+.twitch-chat-sender {
+	background: white;
+	padding-left: 12px;
+	padding-right: 12px;
+	opacity: 0;
+	animation: bubble-enter 1500ms ease-in-out forwards 600ms;
+}
+
+.twitch-chat-message > :is(mark, img) {
+	animation: content-enter 1500ms ease-in-out forwards 600ms;
+}
+
+.twitch-chat-message > mark {
```

Comment:
We currently use the `<mark>` tag two different ways, because I'm a monster 😂

1. `<mark data-twitch-mentioned-user>`, for when someone is @-tagged
2. `<mark class="twitch-chat-command>`, for when `showCommands` is true and someone uses an `!` command.

I should probably go in and add a `data-twitch-chat-command-inner` like I did for the sender… But regardless!

If you want both the `@mentions` and the `!commands` to look different, you might want to break this up into more specific rules. Otherwise, we can keep this!